functions/index.js

```diff
@@ -136,7 +148,7 @@ exports.joinGame2 = functions.https.onCall((data, context) => {
   // Grab the text parameter.
   const id = data.id;
   const username = data.username;
-  const league = 1;
+  const league = 1 - 1;
```

Comment:
1-1?

Reply:
It's just for me to not forget that when the pull request passes I need to put `league - 1` and update the script accordingly